paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -295,13 +311,6 @@ internal class CustomerSheetViewModel @Inject constructor(
         }
     }
 
-    override fun onCleared() {
-        _result.update {
-            null
-        }
-        super.onCleared()
-    }
```

Comment:
What’s up with this change?

Reply:
This is no longer required! The viewModel onCleared will not get called when the activity finishes.

Reply:
I definitely will go back and fix the activity view model lifecycle as it is not correct.